Sources/GRPC/GRPCServerCodec.swift

```diff
@@ -75,8 +78,9 @@ extension GRPCServerCodec: ChannelOutboundHandler {
 
     case .message(let message):
       do {
-        let messageData = try message.serializedData()
-        context.write(self.wrapOutboundOut(.message(messageData)), promise: promise)
+        var responseBuffer = context.channel.allocator.buffer(capacity: protobufMetadataSize)
+        try messageWriter.write(message, into: &responseBuffer)
```

Comment:
nit: we use explicit `self.` everywhere (it makes things easier to review since it's more obvious where something is coming from)